cmake/FindGMP.cmake

```diff
@@ -87,6 +87,23 @@ if(NOT GMP_FOUND_SYSTEM)
 
   set(CONFIGURE_OPTS "")
   set(CONFIGURE_ENV "")
+  
+  if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    # GMP yields the following message at the end of the build process.
+    #     WARNING: `makeinfo' is missing on your system.
+
+    # This is a specific issue to Github CI on linux environments:
+    #     https://github.com/ps2dev/ps2toolchain/issues/64
+    #     https://github.com/spack/spack/issues/34906
+    #     https://github.com/periscop/candl/issues/16
+    #     https://github.com/microsoft/vcpkg/issues/22671
+    # Many solution attempts have been tried, but none worked.
+
+    # Since makeinfo just builds the documentation for GMP, 
+    # it is possible to get around this issue by just disabling it:
+    set(CONFIGURE_ENV env "MAKEINFO=true")
+  endif()
```

Comment:
If this is just specific to CI we should add this fix to the CI workflow instead of the build system.

Reply:
I tested in a local ubuntu machine and the issue did not happen. 
Yes. I will do that.